### DISCUSSION TITLE
テストコード内では `var` (型推論) を使わないほうがいいのか？

### DISCUSSION THREAD

**Discussion by murnana** [code-polaris/reading-circle-tdd#54] [Q&A]
第8章を進めているときに、Javaではコンパイルエラーになりますが、
C#等、型推論でローカル変数の宣言が行われる言語の場合、
型推論によって、関数の返却が Dollerのままだと、
`Money` クラスに `Times` がいないことに気づきにくいかと思います (何らかのエラーにもならないので)

なので、今回のように戻り値に指定がある場合は
型推論で書かないほうがいいのかな…？とか思いました。

普段テストコードを書かないので、一般的な意見を聞きたいですー。

**Comment by shino51** [comment]
確かにそうですね～。でも、Javaでも普通に five.times(2) みたいに戻り値をアサインしなくてもエラーにならないので、一緒のことかなぁとは思います…！
